### ISSUE TITLE
Our procedure `cfg_bids` does not work with new datalad version because `cfg_metadatatypes` is deprecated 

### ISSUE THREAD

**Issue by celprov** [TheAxonLab/hcph-sops#465]
In the datalad repository, it is announced that [`cfg_metadatatypes` is deprecated](https://github.com/datalad/datalad/blob/429c8cfe6836de1ad1d7a50c5085c627b7b2bef2/docs/source/changelog.rst?plain=1#L668-L672).

As such, that procedure is not installed in ${PYTHON_SITE_PACKAGES}/datalad/resources/procedures/ as it should.

I have datalad version 0.19.4 installed.

**Comment by celprov** [comment]
A work around is to create a symbolic link of the procedure from previous version of Datalad:
```
git clone git@github.com:datalad/datalad.git
cd datalad
git checkout pypi/0.17.6
PYTHON_SITE_PACKAGES=$( python -c 'import sysconfig; print(sysconfig.get_paths()["purelib"])' )
ln -s <path_to_code>/datalad/datalad/resources/procedures/cfg_metadatatypes.py ${PYTHON_SITE_PACKAGES}/datalad/resources/procedures/
```

**Comment by oesteban** [comment]
Solved in #471.

**Comment by oesteban** [comment]
(requiring datalad >= 1.0)
